rdr_service/tools/tool_libs/backfill_coreminuspmtime.py

```diff
@@ -0,0 +1,77 @@
+from rdr_service import clock
+from rdr_service.dao.participant_summary_dao import ParticipantSummaryDao
+from rdr_service.logic.enrollment_info import EnrollmentCalculation, EnrollmentDependencies
+from rdr_service.model.participant_summary import ParticipantSummary
+from rdr_service.participant_enums import EnrollmentStatus
+from rdr_service.repository.questionnaire_response_repository import QuestionnaireResponseRepository
+from rdr_service.services.system_utils import list_chunks, min_or_none
+from rdr_service.tools.tool_libs.tool_base import cli_run, ToolBase
+
+tool_cmd = 'backfill-coreminuspmtime'
+tool_desc = 'Backfill CoreMinusPMTime values that had been incorrectly updated'
+
+
+class BackfillCoreMinusPMTime(ToolBase):
+    def run(self):
+        super().run()
+        with self.get_session() as session:
+            summary_list = session.query(ParticipantSummary).all()
+            summary_dao = ParticipantSummaryDao()
+
+            chunk_size = 50
+            for summary_list_subset in list_chunks(lst=summary_list, chunk_size=chunk_size):
+                for summary in summary_list_subset:
+                    calculated_date = self.calculate_core_minus_pm_time(summary, session)
+                    if calculated_date:
+                        if calculated_date < summary.enrollmentStatusCoreMinusPMTime:
```

Comment:
readability suggestion:
```suggestion
                    if calculated_date and calculated_date < summary.enrollmentStatusCoreMinusPMTime:
```